### DISCUSSION TITLE
File input component

### DISCUSSION THREAD

**Discussion by jknn5046** [illacloud/illa-builder#729] [Ideas]
I am wondering if it is possible to add file input componet, so that user can upload to server.
For example, if I want to upload a file via a POST Restful API, I would need a file selecter to pick the file from my computer, so taht my pre-defined Restful API could send the file. Or, maybe I want to preview the data of an excel or csv file in a Table component.

I hope these features could be added:
1. File select component
2. Select value type(text, file, etc.) when editing Restful API body

Thank you.

**Comment by donqu1xotechina** [comment]
Hey There：

Thank you for using our product. We are developing the FIle Select Component at this moment, it should be done soon. We are also wondering in what situation you would like to use the Select Value type, could you please elaborate more detail? You can also join our Discord Community for more efficient communication.

Again, we appreciate you sharing your thoughts about our product.

Best,

ILLA Tea,
